encodings/fastlanes/src/bitpacking/compute/slice.rs

```diff
@@ -140,4 +152,24 @@ mod test {
         assert_eq!(doubly_sliced.offset(), 639);
         assert_eq!(doubly_sliced.len(), 784);
     }
+
+    #[test]
+    fn slice_empty_patches() {
+        // We create an array that has 1 element that does not fit in the 6-bit range.
+        let array =
+            BitPackedArray::encode(PrimitiveArray::from((0u32..=64).collect_vec()).array(), 6)
+                .unwrap();
+
+        assert!(array.patches().is_some());
+
+        let patch_indices = SparseArray::try_from(array.patches().unwrap())
```

Comment:
perhaps we should just return `SparseArray` directly from BPA::patches. I know we always return Array, but turning it into a SparseArray is pretty much the first thing we always do with it

Reply:
eh, I think we should handle non sparse patches. I will work on that